README.md

```diff
@@ -126,8 +128,15 @@ You can use any of these services to host your bundles:
 * [AWS S3](https://aws.amazon.com/s3/)
 * [Google Cloud Platform](https://cloud.google.com/)
 * [Netlify](https://www.netlify.com/)
+* [Files API](#deploying-using-files-api)
 * ...
 
+### Deploying using Files API
+
+EVRYTHNG platform provides powerful cloud file system, or Files API, which could be used
+to store your custom components. Files API requires an API key with valid permissions to be provided from environment. It could be done either with `EVT_AUTH` environment variable or `~./evt-auth` file.
```

Comment:
do not force people to dirty their home environment.

1) use env variable; or allow people to pass the key in the arguments
2) what would be the structure of the file? if there's only one variable, there's no need for a file. also, we may have a proper credentials/profiles/configuration file in the future, so we better not define such a structure now

(path was also wrong: `~/.evt-auth`)